src/main/types/SIOP.types.ts

```diff
@@ -91,10 +91,18 @@ export interface AuthenticationResponsePayload extends JWTPayload {
   did: string;
   registration?: DiscoveryMetadataPayload;
   registration_uri?: string;
-  vp?: VerifiablePresentation;
+  verifiable_presentations?: VerifiablePresentationWrapper[];
```

Comment:
This is not correct. According to https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html#name-siop-with-vp_token we should have a vp_token here

Reply:
The above explicitly is about how it should look like in SIOP for both the request and response